manticore/ethereum.py

```diff
@@ -389,17 +389,25 @@ def simplify(x):
             value = arithmetic_simplifier(Operators.CONCAT(size, *map(Operators.ORD, data[offset + padding:offset + padding + byte_size])))
             return simplify(value)
 
-        if ty == u'uint256':
-            return get_uint(256, offset), offset + 32
-        elif ty in (u'bool', u'uint8'):
+        if ty.startswith('uint') and 0 <= int(ty[4:]) <= 256:
+            size = int(ty[4:])
```

Comment:
It might not matter that much but it might be good idea to calculate `int(ty[4:])` just once.

Reply:
Uhm. Any suggestion on how to do that? 
I think I will need to split the two checks: `ty.startswith('uint')` and  `0 <= int(ty[4:]) <= 256` to do that, so the code will be slighter more complicated.

Reply:
Not really, seems like we have to split those :(